[29.x] qt: 29.1 translations update

hebasto opened the pull request:
This PR fetches the recent translation updates from Transifex.

Closes https://github.com/bitcoin/bitcoin/issues/32295.

**Notes for reviewers:**

1. "fr_CM" and "fr_LU" have been dropped as part of [phasing out of territory-specific translations](https://app.transifex.com/bitcoin/communication/d:402657d1-6254-4ce9-8d26-e7827652c627/?q=project%3Abitcoin).

2. The translations for the following languages, which appear to be the result of a mistake or an act of vandalism, have been discarded (also see https://github.com/bitcoin/bitcoin/pull/32004):
    - Czech (cs)
    - Danish (da)
    - Dutch (nl)
    - Vietnamese (vi)

3. Update for Silesian (szl) has been discarded as malicious.

hebasto committed:
qt: 29.1 translations update

hebasto commented:
> Closes #32295.

However, the Polish (pl) translation still have a couple of issues:

1. String 160: Mismatch between 'S&how' and '%Wyświetl'

2. String 452: Mismatch between 'Transaction broadcast successfully! Transaction ID: %1' and 'transakcja'

jesterhodl commented:
> Transaction broadcast successfully

Both fixed in Transifex.

hebasto commented:
> > Transaction broadcast successfully
> 
> Both fixed in Transifex.

Thanks! Updated and undrafted.

jesterhodl commented:
So I rerun the script and found some more issues and updated Transifex.
v29 is not translated 100% yet from what I see, but at least the worst offenders: spam, typos, gross errors should be fixed now.

hebasto commented:
> So I rerun the script and found some more issues and updated Transifex. v29 is not translated 100% yet from what I see, but at least the worst offenders: spam, typos, gross errors should be fixed now.

Thank you! Updated.

jesterhodl commented:
> The erroneous format specs still remain: [#32295 (comment)](https://github.com/bitcoin/bitcoin/issues/32295#issuecomment-2827238571)

Ok, for Polish there was one. I just corrected it.
I'm happy to correct other languages for this sort of issue, but I'm only granted access to Polish team in Transifex.
If someone grants me temporary access I can fix others and go back to being just Polish Team.

jesterhodl commented:
check_translations shouldn't complain for any polish labels as of now. Transifex is updated.

jesterhodl commented:
> > check_translations shouldn't complain for any polish labels as of now. I ran it.
> 
> Thanks. Just for reference, the script doesn't _need_ to report no complaints. Sometimes it complains about perfectly fine translations. Also, it may not even be deterministic if called several times with a cleared cache.

OK. By the way I noticed it. It complained A should be B, and then B should be A. Neither form would make it happy. I opted for something else.

hebasto commented:
Fetched the updated Polish (pl) translation. Thanks to @jesterhodl !

@maflcko's [comment](https://github.com/bitcoin/bitcoin/pull/32352#issuecomment-2833289993) has also been addressed.

I believe that any further translation improvements should be made through communication with the language coordinators, which I'm currently trying to arrange.

jesterhodl commented:
Going forward:
- who do I contact if there's an issue with source labels?
- do I have to do anything when I just maintain new or updated labels? eg. when I align wording across the app

hebasto commented:
> Going forward:
> 
>    * who do I contact if there's an issue with source labels?

Feel free to ping me.

> 
>    * do I have to do anything when I just maintain new or updated labels? eg. when I align wording across the app

Mind clarifying please?

hebasto commented:
> > > * do I have to do anything when I just maintain new or updated labels? eg. when I align wording across the app
> > 
> > 
> > Mind clarifying please?
> 
> Sure. I translate lots of apps on Transifex, Weblate, etc. I routinely revisit to check for new things to update or correct existing translations based on my experience or feedback. So I update the transifex translations.
> 
> When I do this, is it sufficient and github pulls the .ts files automatically, or would it be wise to ping someone?

Yes, that is sufficient, provided no one else has modified your translations. In that case, marking translations as reviewed might help.

laanwj commented on src/qt/locale/bitcoin_es.ts:
```diff
@@ -165,19 +165,19 @@ Solo es posible firmar con direcciones de tipo "legacy".</translation>
     <message>
         <source>Warning: If you encrypt your wallet and lose your passphrase, you will &lt;b&gt;LOSE ALL OF YOUR BITCOINS&lt;/b&gt;!</source>
-        <translation type="unfinished">Atención: Si cifras el monedero y pierdes la frase de contraseña, &lt;b&gt;¡PERDERÁS TODOS TUS BITCOINS&lt;/b&gt;!</translation>
+        <translation type="unfinished">Atención: Si cifras el monedero y pierdes la frase de contraseña, &lt;b&gt;¡PERDERÁS TODOS TUS BITCOIN&lt;/b&gt;!</translation>
```
First thought this change was a bit strange, but "lose all your bitcoin" singular might make more sense even in English?

laanwj commented on src/qt/locale/bitcoin_es.ts:
```diff
@@ -1148,7 +1176,7 @@ El proceso de migración creará un respaldo del monedero antes de migrar. Este
     </message>
     <message>
         <source>Are you sure you wish to close the wallet &lt;i&gt;%1&lt;/i&gt;?</source>
```
Another translation with HTML in it.

hebasto replied:
This comes from:https://github.com/bitcoin/bitcoin/blob/f490f5562d4b20857ef8d042c050763795fd43da/src/qt/walletcontroller.cpp#L94

hebasto commented on src/qt/locale/bitcoin_bg.ts:
```diff
@@ -1039,7 +1039,7 @@ Signing is only possible with addresses of the type 'legacy'.</source>
         <source>Creating Wallet &lt;b&gt;%1&lt;/b&gt;…</source>
         <extracomment>Descriptive text of the create wallet progress window which indicates to the user which wallet is currently being created.</extracomment>
-        <translation type="unfinished">Създаване на уолет 1 1%1 1</translation>
+        <translation type="unfinished">Създаване на уолет &lt;b&gt;%1&lt;/b&gt;…</translation>
```
This comes from: https://github.com/bitcoin/bitcoin/blob/f490f5562d4b20857ef8d042c050763795fd43da/src/qt/walletcontroller.cpp#L251-L253

laanwj reviewed (approved):
ACK fc60337733a9dffaa42e08fcbff0ab24b5f679a4

i looked over the changes and did some spot checks and found nothing that looks malicious or appears out of the ordinary.
